packages/medusa/src/api/routes/admin/draft-orders/create-draft-order.ts

```diff
@@ -256,7 +255,8 @@ export class AdminPostDraftOrdersReq {
   @Type(() => Item)
   @IsNotEmpty()
   @ValidateNested({ each: true })
-  items: Item[]
+  @IsOptional()
```

Comment:
question: should we keep the ValidatedNested in case it is provided? Also, it is missing `?` as an optional prop

Reply:
The optional `?` is already there, unless you are asking me to add it to another place.

Reply:
sorry I ve missed it in my reading ahah